osquery/tables/system/windows/system_info.cpp

```diff
@@ -0,0 +1,88 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <boost/algorithm/string.hpp>
+
+#include <osquery/sql.h>
+#include <osquery/system.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/conversions.h"
+#include "osquery/tables/system/windows/wmi.h"
+
+namespace osquery {
+namespace tables {
+
+QueryData genSystemInfo(QueryContext& context) {
+  Row r;
+  r["hostname"] = osquery::getHostname();
+  r["computer_name"] = r["hostname"];
+
+  auto qd = SQL::selectAllFrom("cpuid");
+  for (const auto& row : qd) {
+    if (row.at("feature") == "product_name") {
+      r["cpu_brand"] = row.at("value");
+      boost::trim(r["cpu_brand"]);
+    }
+  }
+
+  WmiRequest wmiUUIDReq("Select UUID from Win32_ComputerSystemProduct");
```

Comment:
Do you mind adding this to the `generateUUID` method in `system.cpp` with a new `#elif` to differentiate Windows from Linux?